packages/css/scripts/build.ts

```diff
@@ -0,0 +1,55 @@
+import {pathToFileURL} from 'node:url'
+import path from 'path'
+import fs from 'fs/promises'
+import postcss from 'postcss'
+import autoprefixer from 'autoprefixer'
+
+const SOURCE_DIR = '../react/src'
+const OUTPUT_DIR = 'components'
+
+async function build(): Promise<void> {
+  await fs.mkdir(OUTPUT_DIR, {recursive: true})
+
+  const cssFiles = await findCssFiles(SOURCE_DIR)
+
+  for (const file of cssFiles) {
+    const relativePath = path.relative(SOURCE_DIR, file)
+
+    // removing the module part of the the file name make it cleaner
+    const outputPath = path.join(OUTPUT_DIR, relativePath.replace('.module.css', '.css'))
+
+    await fs.mkdir(path.dirname(outputPath), {recursive: true})
+
+    const css = await fs.readFile(file, 'utf8')
+
+    // we need to process using same postcss config as the react version
+    const result = await postcss([autoprefixer]).process(css, {
+      from: file,
+      to: outputPath,
+    })
+
+    await fs.writeFile(outputPath, result.css)
+  }
+}
+
+async function findCssFiles(dir: string): Promise<string[]> {
+  const files: string[] = []
+  const entries = await fs.readdir(dir, {withFileTypes: true})
+
+  for (const entry of entries) {
+    const fullPath = path.join(dir, entry.name)
+    if (entry.isDirectory()) {
+      files.push(...(await findCssFiles(fullPath)))
+    } else if (
+      entry.name.endsWith('.module.css') &&
+      !entry.name.includes('stories') &&
+      !entry.name.includes('features')
```

Comment:
Maybe this would be a little bit more robust going forward?

```suggestion
      /^\w+\.module\.css$/.test(entry)
```

That matches `abc.module.css` but not `abc.xyz.module.css`. wdyt?

Reply:
Do we need to match `abc.xyz.module.css`? 🤔